components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpoint.java

```diff
@@ -235,6 +235,9 @@ public class OAuth2AuthzEndpoint {
 
     private static final String PARAMETERS = "params";
     private static final String FORM_POST_REDIRECT_URI = "redirectURI";
+    private static final String SERVICE_PROVIDER = "s";
```

Comment:
since these are processed again in the oauth_response file, shall we add descriptive param names?
eg: serviceProvider, userTenantDomain, TenantDomain, etc

Reply:
This was resolved with commit 15b53ca30d1b06ecd9cc4a26581af85e84734456